$MirOS$

	 mpcc -DG_DISABLE_DEPRECATED -O2 -pipe -std=gnu99 -Os -fweb -frename-registers -Wformat -Wstrict-aliasing -Wbounded -fwrapv -fno-align-functions -fno-align-labels -falign-loops=4 -falign-jumps=4 -march=i486 -mpush-args -mpreferred-stack-boundary=2 -fhonour-copts -Wall -c have_mmx.S  -fPIC -DPIC -o .libs/have_mmx.o
	have_mmx.S:45:3: invalid preprocessing directive #Check
	have_mmx.S:60:3: invalid preprocessing directive #OK

--- gdk-pixbuf/pixops/have_mmx.S.orig	Mon Jul 16 19:44:44 2007
+++ gdk-pixbuf/pixops/have_mmx.S	Tue Dec  9 22:07:02 2008
@@ -42,7 +42,6 @@ __pixops_have_mmx:
 	
 	push	%ebx
 
-# Check if bit 21 in flags word is writeable
 
 	pushfl	
 	popl	%eax
@@ -57,7 +56,6 @@ __pixops_have_mmx:
 
 	je .notfound
 
-# OK, we have CPUID
 
 	movl	$1, %eax
 	cpuid
